lib/taurus/qt/qtgui/taurusgui/taurusgui.py

```diff
@@ -832,8 +833,6 @@ def createCustomPanel(self, paneldesc=None):
             if not ok:
                 return
         w = paneldesc.getWidget(sdm=Qt.qApp.SDM, setModel=False)
-        if hasattr(w, 'setCustomWidgetMap'):
-            w.setCustomWidgetMap(self.getCustomWidgetMap())
```

Comment:
shouldn't we also update the map in this case?

I think a similar code to that of `applyConfig` (i.e., setting a map that is obtained from updating the widget's with the windows' ) should also be done here instead of removing it. ... or am I missing something?